docusaurus/docs/advanced-configuration.md

```diff
@@ -3,21 +3,17 @@ id: advanced-configuration
 title: Advanced Configuration
 ---
 
-The `.config/` directory holds the preferred configuration for the different tools used to develop, test, and build a plugin.
+The plugin `.config/` directory holds the preferred configuration for the different tools used to develop, test, and build a Grafana plugin. Although you can make changes, we recommend against doing so. Instead, follow the guidance in this topic to customize your tooling configs.
 
 :::danger
 
-To make future updates easier do **not** edit files in the `.config` directory. Instead, follow the directions in this article to customise the tooling configurations.
+Customize the plugin tooling at your own risk. If you choose to extend the tooling configurations, then you may experience issues such as failure to compile or load in Grafana.
```

Comment:
I would probably keep the mention of the `.config` folder in this heads-up message. The key point we would like to make is that ideally they shouldn't touch the config files under `.config/`, however they are free to customize them in a bit safer way by following our extensions below.

Reply:
Fixed.